pages/about/groups/agwg.md

```diff
@@ -33,67 +34,80 @@ Some information on this page is also shown on, and may be more current in, the
 
-* [How to contribute to the Working Group and file comments](https://www.w3.org/WAI/WCAG20/comments/);
-* [How to contribute to the source repository directly](https://github.com/w3c/wcag/);
-* [How to participate in (join) the Working Group](/about/groups/agwg/participate/).
+There are many ways you can contribute without being a member of the working group:
 
-## Meetings and Communication
+* **Comment on publications:** During development, the group publishes working drafts for public comment &mdash; these are announced on the [WAI News pages](/news/) and on the [WAI Interest Group mailing lists](/about/groups/waiig/#mailinglist).
+* **Raise, comment on, or propose fixes to WCAG issues:** If you’re aware of an issue with any of the published WCAG 2.x resources, W3C welcomes you to [raise a new issue on GitHub](https://github.com/w3c/wcag/issues/) &mdash; you can also comment on and, even better, propose solutions for existing issues.
+* **Contribute to WCAG 3 discussions:** As WCAG 3 develops, many topics are being discussed &mdash; to contribute your views, see the list of [open WCAG 3 discussions](https://github.com/w3c/wcag3/discussions).
+* **Participate in a community group:** W3C hosts a small number of active community groups that focus on digital accessibility issues &mdash; for details, search for ‘accessibility’ on [Current Groups &mdash; W3C Community and Business Groups](https://www.w3.org/community/groups).
 
-The AG WG conducts its work using a variety of synchronous and asynchronous tools. The [communication page](/about/groups/agwg/communication/) provides details about:
+### Become a participant in the group
 
-* Teleconferences of the Working Group and its task forces (also see [upcoming agendas](/WAI/GL/wiki/Upcoming_agendas), meeting minutes);
-* Face to face meetings (also see face-to-face [meeting pages](https://www.w3.org/WAI/GL/wiki/Meetings));
-* Email lists;
-* [Wiki](https://www.w3.org/WAI/GL/wiki/);
-* [Web-Based Surveys (WBS)](https://www.w3.org/2002/09/wbs/35422/);
-* [WCAG 2 source repository issue tracker](https://github.com/w3c/wcag/issues).
+Joining the AG Working Group enables you to participate fully in the development of the work and influence the deliverables. You and your organization will also be listed as contributors, where appropriate.
 
-These tools are used by participants of the Working Group. For ways non-participants can contribute, see [How to contribute to the Working Group and file comments](https://www.w3.org/WAI/WCAG20/comments/).
+To become a participant requires a commitment from you to support the work of the group in the following ways:
 
-### Meeting Minutes
+* Engage actively with other participants in the working group using the group’s communication channels.
+* Keep up with weekly tasks and the progress of the work &mdash; for example, via the minutes of past meetings, mailing list discussions, and GitHub issue comments.
+* Give your input on time.
```

Comment:
Hmm... seems peculiar. Not a biggie though I guess.

Reply:
Did you mean that 'on time' seems peculiar? 
'In a timely manner' comes across as a little formal. I was looking for a plain language alternative. 
Would 'promptly, when it's needed' be better?

Reply:
Yes, 'give your input on time' seems too informal. Solution is fine